### PR TITLE
Clean convert module

### PR DESCRIPTION
**Pull request by harry093**
Cleaned up convert module. Some functions that I thought we could remove are not able to be removed as they are used in tests. DNACoord and read_dna are now in the new fileio module.

We should look at fixing tests so that special code is not needed solely for testing purposes.

### CODE REVIEW

File: geodepy/fileio.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+
+"""
+Geoscience Australia - Python Geodesy Package
+Constants Module
```
Comment (BatchelorJ) [inline geodepy/fileio.py:5]: Small comment - update comment text to 'File Input/Output Module'

File: geodepy/fileio.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
```
Comment (BatchelorJ) [inline geodepy/fileio.py:1]: Should this just be part of the inputoutput module? Or should we merge the inputoutput module into fileio?

File: geodepy/fileio.py (hunk 1 above)
Comment (harry093) [reply geodepy/fileio.py]: Fixed. Pushing changes now

File: geodepy/fileio.py (hunk 2 above)
Comment (harry093) [reply geodepy/fileio.py:1]: I don't know. I wanted to keep it separate for now. I need to have a look at what's in inputoutput. It depends on pandas and I'm wary of making too many dependencies. I think an external package has to offer a significant benefit to warrant inclusion.

**Comment by harry093** [comment]
@BatchelorJ yeah, all of the PEP8 changes are just making line lengths less than 80 characters

File: geodepy/fileio.py (hunk 2 above)
Comment (BatchelorJ) [reply geodepy/fileio.py:1]: Ok that's fine, something for us to look at and possibly flag as an issue.
Merge away!

**Pull request merged**